internal/cli/get.go

```diff
@@ -221,34 +231,36 @@ func (o *GetOptions) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("creating client: %w", err)
 	}
 
-	kind, name, err := parseAndValidateKindName(args[0])
+	kind, names, err := parseAndValidateKindNameFromArgs(args)
 	if err != nil {
 		return err
 	}
 
 	formatter := display.NewFormatter(display.OutputFormat(o.Output))
-	if name == "" {
+
+	// Handle list case (no specific names)
+	if len(names) == 0 {
 		if err := o.handleList(ctx, formatter, clientWithResponses, kind); err != nil {
 			return fmt.Errorf("listing %s: %w", plural(kind), err)
 		}
-	} else {
-		if err := o.handleSingle(ctx, formatter, clientWithResponses, kind, name); err != nil {
-			return fmt.Errorf("reading %s/%s: %w", kind, name, err)
-		}
+		return nil
 	}
```

Comment:
This changes the behavior that getting a single resource no longer returns that resource (e.g. `Device`) but a collection of resources (e.g. `DeviceList`) with a single element.

Reply:
fixed